appserver/tests/payara-samples/classloader-data-api/src/main/java/fish/payara/samples/classloaderdata/InstanceCountService.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package fish.payara.samples.classloaderdata;
+
+/**
+ *
+ * @author cuba
+ */
+public class InstanceCountService {
```

Comment:
Not sure this is necessary at all

Reply:
I still see no point in this class, it does nothing.
InstanceCountTracker can be used directly.